dspace/config/spring/api/virtual-metadata.xml.openaire4

```diff
@@ -0,0 +1,448 @@
+        publicationVolume.volumeNumber = Even more generic volume
+     Then the original item will have, in this case:
+        publicationVolume.volumeNumber = Generic volume
+        publicationVolume.volumeNumber = Even more generic volume
+     Note that these metadata fields have the same signature of schema, element and qualifier, but that poses no problem
+     -->
+    <bean class="org.dspace.content.virtual.Collected" id="issueVolume_volume">
+        <property name="fields">
+            <util:list>
+                <value>publicationvolume.volumeNumber</value>
+            </util:list>
+        </property>
+    </bean>
+    <!--    This Related bean defines a relationship type that will be used on a journal issue.
+            The journal issue will receive 2 metadata fields if we look at the map 'isJournalVolumeOfIssueMap' it'll
+            receive the publicationVolume.volumeNumber metadata field and the creativeworkseries.issn metadata field.
+         Since the creativeworkseries.issn metadata field can't be filled in through the metadata of the journalvolume
+         alone, we need to pass this call higher up the chain to the journalvolume's relationships to find the journal.
+         Therefore we've created this Related bean which will look at the journalvolume's relationships with type
+         'isJournalOfVolume' to find the journal that this journalvolume is related to and it'll then call the next
+         VirtualMetadataConfiguration with that journal to retrieve the value. This will leave the retrieving
+         of the values to the next bean in the chain -->
+    <bean class="org.dspace.content.virtual.Related" id="volumeJournal_issn_related">
+        <property name="relationshipTypeString" value="isJournalOfVolume"/>
+        <property name="place" value="1"/>
+        <property name="virtualMetadataConfiguration" ref="volumeJournal_issn"/>
+    </bean>
+
+    <bean class="org.dspace.content.virtual.Related" id="volumeJournal_title_related">
+        <property name="relationshipTypeString" value="isJournalOfVolume"/>
+        <property name="place" value="1"/>
+        <property name="virtualMetadataConfiguration" ref="volumeJournal_title"/>
+    </bean>
+
+    <util:map id="isJournalOfVolumeMap">
+        <entry key="creativeworkseries.issn" value-ref="volumeJournal_issn"/>
+        <entry key="journal.title" value-ref="volumeJournal_title"/>
+    </util:map>
+    <bean class="org.dspace.content.virtual.Collected" id="volumeJournal_issn">
+        <property name="fields">
+            <util:list>
+                <value>creativeworkseries.issn</value>
+            </util:list>
+        </property>
+    </bean>
+    <bean class="org.dspace.content.virtual.Collected" id="volumeJournal_title">
+        <property name="fields">
+            <util:list>
+                <value>dc.title</value>
+            </util:list>
+        </property>
+    </bean>
+
+    <util:map id="isVolumeOfJournalMap">
+    </util:map>
```

Comment:
This looks odd, an empty mapping?

Reply:
I've just kept what existed in the default virtual-metadata.xml file - https://github.com/DSpace/DSpace/blob/master/dspace/config/spring/api/virtual-metadata.xml#L192

For OpenAIRE4 we don't require Journals or Volumes or Issues and some relations. We can remove all of that. But if this configuration replaces the default file, already existent Entities will loose the display of that data.

Reply:
I would prefer to keep this an openaire config file (also mentioned in https://github.com/DSpace/DSpace/pull/2592#discussion_r359338146) and remove these parts